internal/model/mocks/trace.go

```diff
@@ -15,6 +15,9 @@ import (
 type Trace struct {
 	MockTimeNow func() time.Time
 
+	MockOnDNSRoundTripForLookupHost func(
+		started time.Time, reso model.Resolver, query model.DNSQuery, response model.DNSResponse,
+		addrs []string, err error, finished time.Time)
```

Comment:
```suggestion
		addrs []string, err error, finished time.Time)

```

For aesthetic reasons I'd like to have an empty line here